templates/registration/login.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base.html" %}
+{% block content %}
+  <h3>Login</h3>
+  <form action="{% url "taxi:login" %}" method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <button type="submit">Login</button>
+
+    {% if form.errors %}
+      <div>
+        <strong>Error!</strong>
+        <ul>
+          {% for field in form %}
+            {% for error in field.errors %}
+              <li>
+                errors
```

Comment:
The placeholder 'errors' should be replaced with the actual error message. Use `{{ error }}` to display the error message for each field.